spacy_experimental/multiembed/tests/test_multiembed.py

```diff
@@ -0,0 +1,151 @@
+import os
+import math
+
+import pytest
+import spacy
+import srsly
+
+from spacy.training import Example
+from spacy import util
+from typing import List
+from spacy.tokens import Doc
+from thinc.api import Config
+
+
+multiembed = spacy.registry.architectures.get(
+    "spacy-experimental.MultiEmbed.v1"
+)
+set_attr = spacy.registry.misc.get(
+    "spacy-experimental.set_attr.v1"
+)
+
+TEST_TEXTS = [
+    "Hello my name is Bob",
+    "I know right?",
+    "I'm just an example"
+]
+TEST_LABELS = [
+    {"cats": {"greeting": 1.0}},
+    {"cats": {"question": 1.0}},
+    {"cats": {"statement": 1.0}}
+]
+
+
+cfg_string_textcat = """
+    [nlp]
+    lang = "en"
+    pipeline = ["tok2vec","textcat"]
+
+    [components]
+
+    [components.textcat]
+    factory = "textcat"
+
+    [components.textcat.model]
+    @architectures = "spacy.TextCatCNN.v2"
+    exclusive_classes = true
+
+    [components.textcat.model.tok2vec]
+    @architectures = "spacy.Tok2VecListener.v1"
+    width = ${components.tok2vec.model.encode.width}
+
+    [components.tok2vec]
+    factory = "tok2vec"
+
+    [components.tok2vec.model]
+    @architectures = "spacy.Tok2Vec.v2"
+
+    [components.tok2vec.model.embed]
+    @architectures = "spacy-experimental.MultiEmbed.v1"
+    width = ${components.tok2vec.model.encode.width}
+    attrs = ["ORTH", "SHAPE"]
+    include_static_vectors = false
+    unk = 0
+
+    [components.tok2vec.model.encode]
+    @architectures = "spacy.MaxoutWindowEncoder.v2"
+    width = 96
+    depth = 1
+    window_size = 1
+    maxout_pieces = 2
+
+    [initialize]
+
+    [initialize.before_init]
+    @callbacks = "set_attr.v1"
+    path = tables.msg
+    component = "tok2vec"
+    layer = "multiembed"
+    attr = "tables"
+    """
+
+
+def _get_examples():
+    nlp = spacy.blank("en")
```

Comment:
`nlp` should be passed in so you're using the same vocab for the examples as the model.